frontend/packages/console-demo-plugin/src/plugin.tsx

```diff
@@ -110,6 +110,27 @@ const plugin: Plugin<ConsumedExtensions> = [
       landingPageURL: '/search',
     },
   },
+  {
+    type: 'NavItem/Href',
+    properties: {
+      perspective: 'test',
+      componentProps: {
+        name: 'Test Search',
```

Comment:
Don't we generally want to have all nav items placed under sections that can be expanded or collapsed?

If dev-console needs top-level links (without a section), then the answer would be no.

But if the answer is yes, I'd suggest to keep `section` as a mandatory property in `NavItem`.

Reply:
In dev console we do not have every item within a section. This was a decision made by UX.

Reply:
OK, understood.